sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -449,6 +466,9 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
    * an effect, a directory will still navigate.
    */
   async _handleDoubleClicked(e: ItemClickEvent) {
+    if (this._fetching) {
+      return;
```

Comment:
Same concern as above. The page sometimes auto-refreshes, right? Seems like this will make the UI feel like it randomly doesn't work.
Likewise for all the other places where the UI is going to give me the cold shoulder and silently ignore what I asked it to do.